idaes/ui/fsvis/static/css/main.css

```diff
@@ -335,4 +335,13 @@ input:checked + .slider:before {
 
 .dropdown-item:active {
   background-color: white;
+}
+
```

Comment:
Please add some comments here saying what these affect and why. I would suggest more specific class names like "streamtable-variable"